feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -0,0 +1,79 @@
+# gNMI-1.19: gNMI subscribe with sample mode for backplane capacity counters
+
+## Summary
+WBB is required to support gNMI Subscribe with sample mode for various counters.
+This test if to verify that DUT supports gNMI Subscribe with sample mode, updating
+the available backplane capacity counters correctly while forwarding traffic
+
+*   Get backplace capacity before any traffic is sent
+
+*   Send some traffic and wait for the sample to be collected
+
+*   Send more traffic and wait for the sample to be collected
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_9.testbed
```

Comment:
This is expensive to reserve 9 IXIA ports. Can this test be done with just 2 ports?
I do not know. Just asking.

Reply:
To see changes to backplane capacity which is in terabits, I think we will need more than 2 ports. I also intend to check if the counters are valid so I though of consuming as much as possible.